spec/support/notify_fake_client.rb

```diff
@@ -0,0 +1,5 @@
+class NotifyFakeClient
```

Comment:
One thought which occurs with this is testing in a wider context? Ie Cucumber, functional

We could do with some equivalent of a `test` delivery method? Maybe this could store an array of sent emails. Could the main notify class be changed to have a mode where by it always uses this when `Rails.env.test?`

Reply:
Yes, that's a really good idea and will be easy to implement when we're on the next stage and hooking things together. I'm not so sure about making the class behave differently depending on env but am happy to explore it as an idea.